packages/node_modules/@webex/plugin-meetings/src/participants/index.js

```diff
@@ -0,0 +1,105 @@
+/*!
+ * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {isEmpty} from 'lodash';
+import {StatelessSparkPlugin} from '@ciscospark/spark-core';
+
+import {MEETINGS, EVENT_TRIGGERS} from '../constants';
+
+import ParticipantCollection from './collection';
+import ParticipantRequest from './request';
+import ParticipantUtil from './util';
+
+/**
+ * @class Participants
+ */
+export default class Participants extends StatelessSparkPlugin {
+  namespace = MEETINGS;
+
+  /**
+   *
+   * @param {Object} attrs
+   * @param {Object} options
+   * @memberof Participants
+   */
+  constructor(attrs, options) {
+    super({}, options);
+    this.participantRequest = new ParticipantRequest({}, options);
+    this.locusUrl = null;
```

Comment:
add a TODO to update all the modules with a locusUrl change and add a updateLocusUrl function